pkg/webhook/workspace/handler/workspace.go

```diff
@@ -26,7 +26,7 @@ import (
 // StopStartDiffOption is comparing options that should be used to check if there is no other changes except changing started
 var StopStartDiffOption = []cmp.Option{
```

Comment:
Does it makes sense to rename that DiffOptions? It does not seem to be startStop anymore

Reply:
Fixed